docs/_guides.yaml

```
@@ -17,6 +17,8 @@ toc:
     path: /docs/genkit/
   - title: Get started
     path: /docs/genkit/get-started
+  - title: Migrate from Genkit 0.5
```

Migrate to Genkit 0.9?
Upgrade to Genkit 0.9?

```suggestion
  - title: Migrate to Genkit 0.9
```

---

or maybe it's fine this way.... feel free to ignore.